lib/game/widgets/tap_to_jump_overlay.dart

```diff
@@ -1,5 +1,12 @@
 import 'package:flutter/material.dart';
 import 'package:super_dash/l10n/l10n.dart';
+import 'dart:io';
+import 'package:flutter/foundation.dart';
+
+bool get isAndroid => !kIsWeb && Platform.isAndroid;
+bool get isIOS => !kIsWeb && Platform.isIOS;
+bool get isWindows => !kIsWeb && Platform.isWindows;
```

Comment:
why add `!kIsWeb` in this condition?

Reply:
because it is possible that people open the app in browser but on a android or iOs or windows, opening the tab as in 'desktop view'. So, I believed that adding this condition would help.